### PR TITLE
add reload in useQuery

### PR DESCRIPTION
**Pull request by Ashniu123**

### CODE REVIEW

**Comment by aravindet** [comment]
Thanks for the tests man, looks like a little linting issue exists: components need to have names.

File: src/react/react.test.js [hunk 1]
```diff
@@ -0,0 +1,37 @@
...
+  let backend;
+
+  beforeEach(() => {
+    g = new Graffy();
+    backend = mockBackend();
+    backend.read = jest.fn(backend.read);
+    g.use(backend.middleware);
+    wrapper = ({ children }) => (
+      <GraffyProvider store={g}>{children}</GraffyProvider>
+    );
```
Comment (aravindet) [inline src/react/react.test.js:20]: Perhaps

```suggestion
    wrapper = function Wrapper ({ children }) {
      return (
        <GraffyProvider store={g}>{children}</GraffyProvider>
      );
    };
```

will fix the lint error?

Comment (Ashniu123) [reply src/react/react.test.js:20]: Throws `no-unused-vars` error.

To resolve this, can we please disable this rule?

Comment (aravindet) [reply src/react/react.test.js]: > To resolve this, can we please disable this rule?

Generally no.

If we legitimately need an unused variable in our code (but why would we?) we should add a `// eslint-disable-next-line no-unused-vars` comment and a separate comment explaining why this unused var is required.

Comment (aravindet) [reply src/react/react.test.js]: Alternately you can prefix unused variable names with an `_` (e.g. `_Wrapper`) to avoid the unused var error.

**Comment by Ashniu123** [comment]
Failed test case not associated with my changes. Request to merge.

**Pull request merged**